### DISCUSSION TITLE
After updating to 2024.8.4  all sleeps are marked as off-line.

### DISCUSSION THREAD

**Discussion by bodosom** [teslamate-org/teslamate#3751] [Q&A]
I'm running the 3672 preview on a model year 2022 Model Y in the US. After updating to 2024.8.4 on 18-Mar-2024 (see image below) all sleeps are marked as off-line (see image below).

I'm running the 3672 preview.
![SCR-20240320-jzje](https://github.com/teslamate-org/teslamate/assets/17100407/0b329592-263c-415f-b20f-9e143493ba22)

**Comment by cwanja** [comment]
What do you mean by “3627 preview”? 

What are your TelsaMate logs showing? 

Know this came from Tesla Motors Club, seeking more information. Thanks.

**Reply by bodosom** [reply]
https://github.com/teslamate-org/teslamate/pull/3672

image: ghcr.io/teslamate-org/teslamate:pr-3672

**Reply by cwanja** [reply]
You should consider moving back to the main branch as that branch has  been merged into `master` and released. If the `master` release does not resolve your issue, I would comment back on your original post #3658 and tag the @micves for review. Again, logs are going to be needed to troubleshoot.

Going to lock this to prevent duplicate efforts.
